task/inspect-image/0.1/inspect-image.yaml

```
@@ -10,7 +10,7 @@ metadata:
   name: inspect-image
 spec:
   description: >-
-    Get manifest data for source and base images and save them to workspace.
+    The sanity-inspect-image task uses Skopeo to inspect and analyze manifest data of the container's source image, and its base image if available.
```

I forgot about this one too:
```suggestion
    The inspect-image task uses Skopeo to inspect and analyze manifest data of the container's source image, and its base image if available.
```

---

sorry, I forgot too. updated now.